### DISCUSSION TITLE
Support for Unix timestamp in &from and &to url parameters

### DISCUSSION THREAD

**Discussion by dchrno** [datalust/seq-tickets#1316] [Ideas]
We're using Grafana which can link to external websites with a selected interval as URL parameters (panel links). From certain dashboards we'd like to be able to link directly to Seq.

Example:
https://<seq_server>/#/events?signal=signal-m33301&from=${__from}&to=${__to}

For the time period 12.09.2021 00:01:09Z to 12.09.2021 12:01:09Z in Grafana, this expands to:
https://<seq_server>/#/events?signal=signal-m33301&from=1631404869371&to=1631448069371";

Grafana will pass these values as UNIX timestamps which Seq does not understand. Could you consider adding URL support for UNIX timestamps?

C# sample code
if (long.TryParse("1631404869371", out long from_ts))
{
	var from_dt = DateTimeOffset.FromUnixTimeMilliseconds(from_ts).UtcDateTime;
} // yields 12.09.2021 00:01:09, DateTime.Kind=UTC

**Comment by nblumhardt** [comment]
Hi @dchrno - thanks for getting in touch.

It looks like Grafana can already format these as ISO-8601 timestamps; e.g.:

```
https://<seq_server>/#/events?signal=signal-m33301&from=${__from:date}&to=${__to:date}
```

(From https://github.com/grafana/grafana/issues/25396#issuecomment-671516552)

Let me know if this helps!
Nick

**Reply by dchrno** [reply]
Thanks, that was a nice find! This format works fine.

**Reply by nblumhardt** [reply]
Great! 😎
